content/content-strategy/audit-content/analyse-evaluate/example.md

```diff
@@ -2,13 +2,12 @@
 layout: text/callout
 ---
 ### Example
```

Comment:
@sirleech I thought the recommendations from Bri's team were to add visually hidden text to the `Example` heading?